tour-tip/component.js

```diff
@@ -22,22 +22,29 @@ export default class TourTip extends Component {
 	renderCtas (data) {
 		if (data.content.ctas && data.content.ctas.length) {
 			return data.content.ctas.map(cta =>
-						<a href={cta.url} className="tour-tip__cta o-buttons o-buttons--standout" data-trackable="cta">{cta.label}</a>
-				);
+				<a href={cta.url} className="tour-tip__cta o-buttons o-buttons--standout" data-trackable="cta">{cta.label}</a>
+			);
 		}
 	}
 
 	renderImage (data) {
 		if (data.content.imageUrl) {
-			const isSvgExt = data.content.imageUrl.substr(-4,4) === '.svg';
-			const imageServiceOpts = (isSvgExt) ? {format: 'svg'} : {};
+			const imageAttrs = {
+				alt: data.content.imageAlt,
```

Comment:
looks good, might this ever be null? (in which case alt: data.content.imageAlt || '')